R/slab.R

```diff
@@ -1,20 +1,16 @@
-
-
 # default slab function for categorical variables
 # returns a named vector of results
 # this type of function is compatible with aggregate()
-.slab.fun.factor.default <- function(values, cpm) {
+.slab.fun.factor.default <- function(values, cpm, ...) {
```

Comment:
\dots used for optional weights?

Reply:
All of the slab functions now have `...` in their definition for consistency and possible future expansion